src/main/java/io/github/thebusybiscuit/slimefun4/utils/itemstack/ItemStackWrapper.java

```diff
@@ -132,8 +132,8 @@ public void addUnsafeEnchantment(Enchantment ench, int level) {
     public static @Nonnull ItemStackWrapper wrap(@Nonnull ItemStack itemStack) {
         Validate.notNull(itemStack, "The ItemStack cannot be null!");
 
-        if (itemStack instanceof ItemStackWrapper) {
-            return (ItemStackWrapper) itemStack;
+        if (itemStack instanceof ItemStackWrapper itemStackWrapper) {
+            return itemStackWrapper;
```

Comment:
We do not always have to be so literal about it.
```suggestion
        if (itemStack instanceof ItemStackWrapper wrapper) {
            return wrapper;
```